packages/live-status-gateway/src/topics/__tests__/activePlaylist.spec.ts

```diff
@@ -120,7 +132,9 @@ describe('ActivePlaylistTopic', () => {
 				id: 'SEGMENT_1',
 				timing: {
 					expectedDurationMs: 10000,
-					projectedEndTime: 1600000070000,
+					budgetDurationMs: 12300,
+					projectedEndTime: 1600000072300,
+					countdownType: 'segment_budget_duration',
```

Comment:
Since these two behaviors are still desired, I think we should keep to different test cases for them, not replace one with the other.